docs/docsite/rst/rules/default_rules.rst

```diff
@@ -8,51 +8,50 @@ Default Rules
-`E403 <https://github.com/ansible/ansible-lint/blob/master/lib/ansiblelint/rules/PackageIsNotLatestRule.py>`_                                                                                                                               historic                                                                                                                                                                                                                                    Package installs should not use latest                                                                                                                                                                                                      Package installs should use ``state=present`` with or without a version                                                                                                                                                                     
-`E404 <https://github.com/ansible/ansible-lint/blob/master/lib/ansiblelint/rules/RoleRelativePath.py>`_                                                                                                                                     v4.0.0                                                                                                                                                                                                                                      Doesn't need a relative path in role                                                                                                                                                                                                        ``copy`` and ``template`` do not need to use relative path for ``src``                                                                                                                                                                      
-`E405 <https://github.com/ansible/ansible-lint/blob/master/lib/ansiblelint/rules/PackageHasRetryRule.py>`_                                                                                                                                  v4.0.0                                                                                                                                                                                                                                      Remote package tasks should have a retry                                                                                                                                                                                                    Package operations are unreliable as they require network communication and the availability of remote servers. To mitigate the potential problems, retries should be used via ``register: my_result`` and ``until: my_result | success``   
```

Comment:
Minor nit: seems that because the formatting was changed, github shows 11-57 lines as changed, when in reality only line 41 needs to be dropped. Perhaps this is normal and expected but it caught my eye so pointing it out.

Reply:
Thanks for this catch - I think the rst -> html script had not be run since a formatting change in the script. In comparing the output, only 405 has changed so I think this is ok.

Reply:
Ah yes the script changes cell width based on all data, so all lines are altered.